build/ci/release.yml

```diff
@@ -418,7 +418,7 @@ tasks:
           display_name: downloads-center-
       - command: s3.put
         params:
-          role_arn: "arn:aws:iam::119629040606:role/s3-access.cdn-origin-atlascli"
```

Comment:
We migrated mongocli to https://github.com/mongodb/mongodb-cli/tree/main

Reply:
Also all the post packaging tasks are failing after this PR was merged : https://spruce.mongodb.com/version/mongodb_atlas_cli_master_36f21cd8defe25ad9c89cf4d1d09f424e62cff55/tasks?sorts=STATUS%3AASC%3BBASE_STATUS%3ADESC